source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,140 @@ weight: 70
 
-NOTE: The status of the payment will go through several phases until it either
-succeeds or fails. See the [API reference
-section](/api_reference/#api-reference) for more details.
+The status of the payment will pass through several states until it either
+succeeds or fails. See the [__API reference__
+section](/api_reference/#api-reference) of this documentation for more
+details.
 
-## Choosing the return URL and matching user to payment
+## Choose the return URL and match users to payments
 
 For security reasons, GOV.UK Pay does not add the payment ID or outcome to
-your ``return_url`` as parameters.
+your `return_url` as parameters.
+
+You can match users with payments by using a secure cookie, or creating a
+secure random ID:
+
+### Use a secure cookie
 
-To match up a returning user with their payment, there are two recommended
-methods:
+You can use a secure cookie containing the payment ID from GOV.UK Pay. Your
+service should issue this when the payment is created, before sending the user
+to `next_url`. Users will not be able to decrypt a secure cookie, so a
+malicious user could not alter the payment ID and intercept other users’
+payments.
 
-+ use a secure cookie containing the Payment ID from GOV.UK Pay, issued by
-your service when the payment is created (before sending the user to
-``next_url``). Users will not be able to decrypt a secure cookie, so a
-fraudster could not alter the payment ID and intercept other users’ payments.
+### Create a secure random ID
 
-+ create a secure random ID (such as a UUID) and include this as part of the
-``return_url``, using a different ``return_url`` for each payment. Since a
-securely generated UUID is not guessable, fraudsters will not be able to
-intercept users’ payments.
+You can create a secure random ID,such as a UUID, and include this as part of
+the `return_url`. You should use a different ``return_url`` for each payment.
+Securely generated UUIDs are not guessable, so malicious users will
+not be able to securely generated UUID is not guessable, fraudsters will not
+be able to alter the payment ID and intercept other users’ payments.
 
->Note: If you create an ID yourself, you’ll likely need to store this in a
-datastore mapped to the payment ID just after you create a payment.
+If you create an ID yourself, you will likely need to store this in a datastore mapped
+to the payment ID just after you create a payment.
 
 ## Accepting returning users
 
- A user directed to the return URL could have:
+ A user directed to the `return_url` could have:
 
  - paid successfully
- - not paid because their card was rejected or they clicked cancel
- - not paid because your service (via the API) or service manager (via the self-service dashboard) cancelled the payment in progress
+ - not paid because their card was rejected or they selected __Cancel__
+ - not paid because your service (via the API) or service manager (via the GOV.UK Pay admin tool) cancelled the payment in progress
```

Comment:
Just want to check to be 100% sure that 'GOV.UK Pay admin tool' is an acceptable substitute for 'self-service dashboard'.

Reply:
I think it is but I’ve just checked and a payment can’t be cancelled from the admin tool (only the API).